WooCommerce/WooCommerceTests/ViewRelated/Blaze/BlazeEligibilityCheckerTests.swift

```diff
@@ -71,9 +70,8 @@ final class BlazeEligibilityCheckerTests: XCTestCase {
     func test_isEligible_is_false_when_blaze_is_not_approved() async {
         // Given
         stores.authenticate(credentials: .wpcom(username: "", authToken: "", siteAddress: ""))
-        mockDefaultSite()
+        mockDefaultSite(isEligibleForBlaze: false)
```

Comment:
Nit: Should we consider adding a separate test for when Blaze is approved but the user is not an admin? i.e. Test that both `canBlaze` and `isAdmin` need to be true in order to be eligible.

Reply:
Added in https://github.com/woocommerce/woocommerce-ios/pull/10020/commits/9841e1b5151892f10272f24c563f5e8e4852868c